Cards/MadGraph5_aMCatNLO/TT/TTNuNu_amcatnlo-pythia8/TTNuNu_amcatnlo-pythia8.json

```diff
@@ -0,0 +1,23 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "0",
+        "maxjetflavor": "5"
+    },
+    "model_params": "massW.dat",
+    "model_params_user": [
+        "set run_card ptj 20",
```

Comment:
Why do we apply jet pT cuts in a simple NLO sample?

Reply:
ah should remove forgot to do